app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/widget/TimetableWidgetService.java

```diff
@@ -112,14 +112,14 @@ public RemoteViews getViewAt(int position) {
             DateFormat timeFormat = android.text.format.DateFormat.getTimeFormat(applicationContext);
             String time = timeFormat.format(currentItem.getStartTime()
                                                        .getTime());
-            time += "-" + timeFormat.format(currentItem.getEndTime()
+            time += " – " + timeFormat.format(currentItem.getEndTime()
```

Comment:
A en dash. Nice!
having the format string in strings.xml would probably be better, but this should be fine for now

Reply:
Fixed.